prototype/simulator/container/src/destination/destination.js

```diff
@@ -44,9 +46,71 @@ class Destination {
 		this.cognitoUser = usr
 		this.userData.creds = creds
 		this.userData.clientId = `${this.userData.identity}:` + Date.now()
-		this.orderInterval = setInterval(
-			this.sendOrder.bind(this),
-			utils.getMsFromParams(this.params))
+
+		// if event file is not present would generate orders based on the time interval
+		if (!this.params.eventsFilePath) {
+			this.orderInterval = setInterval(
+				this.sendOrder.bind(this),
+				utils.getMsFromParams(this.params),
+			)
+		} else {
+			// otherwise replay the events after waiting 2 seconds
+			setTimeout(
+				this.replayEvents.bind(this),
+				2000,
+			)
+		}
+	}
+
+	async replayEvents () {
+		logger.info('Replaying event file')
+
+		const nowTime = dayjs().format('HHmmss') - 2 // seconds tolerance for the preparation phase
```

Comment:
is `-2` subtracting 2 seconds from the dayjs object?

`.format()` returns a `string`